projects/aws/eks-anywhere-test/Makefile

```diff
@@ -1,5 +1,6 @@
 BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
 GIT_TAG?=$(shell cat GIT_TAG)
+HELM_VERSION=$(GIT_TAG:v%=%)+$(shell git rev-parse --short HEAD)
```

Comment:
So we basically do this already [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L125) for IMAGE_TAG. I wonder if instead of `HELM_VERSION?=$(IMAGE_TAG)-helm` in the Common.mk we could just do `HELM_VERSION?=$(GIT_TAG:v%=%)-$(GIT_HASH)` and then not need to override this?  Do you expect that will be what we do in "most" cases?  I could always continue to be overrideable by individual projects.

Reply:
I see two possible issues with that approach. The first being, that I think parts of the presubmit tasks are expecting the `-helm` suffix. Those can be updated. However, @TerryHowe mentioned something about how the `-helm` might be what distinguishes the helm chart from the docker image that it installs when stored in a registry? If that's true, and I'm understanding correctly, then it's possible that removing the `-helm` suffix would break things there.

The second part is that we have, for better or for worse, upgraded to helm 3.8, and it requires packages to have a semantic version. `HELM_VERSION?=$(GIT_TAG:v%=%)-$(GIT_HASH)` is not a valid semantic version (the `-` would have to change to a `+`). This too might break some other presubmits, but we can probably update those as well.

Reply:
The -helm is just to disambiguate between container images and helm charts. It kind of sounds like we might move helm charts to their own repo, but that would be a topic for another PR